src/screens/Article.js

```diff
@@ -1,72 +1,87 @@
 import React from 'react';
 import {
-  View,
   StyleSheet,
   StatusBar,
   Image,
-  TouchableOpacity,
+  ImageBackground,
 } from 'react-native';
+import { LinearGradient } from 'expo';
+
 // galio components
-import { Text, AuthorSection, NavBar } from '../';
+import { Block, Text, AuthorSection, NavBar } from '..';
+
+import theme from '../theme';
+
+const bgImage = 'https://images.unsplash.com/photo-1533106497176-45ae19e68ba2?fit=crop&w=760&q=80';
 
 const Article = props => (
-  <View style={{ flex: 1 }}>
-    <StatusBar hidden={false} barStyle="light-content" />
-    <Image
-      style={{ width: '100%', height: '45%' }}
-      source={{
-        uri:
-          'https://images.unsplash.com/photo-1533106497176-45ae19e68ba2?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=fc7f17d00c13cf50d62f4ebd5743a9bc&auto=format&fit=crop&w=1950&q=80',
-      }}
-    />
-    <View style={{position: 'absolute', top: 0, left: 0, right: 0}}>
-      <NavBar transparent leftIconColor="#FFF" onLeftPress={() => props.navigation.openDrawer()} />
-    </View>
-    <View style={styles.articleContainer}>
-      <Text h2>I would happily watch a TV show about crabs</Text>
-      <Text p muted style={{ marginTop: 5 }}>
-        Interviewing this super star
-      </Text>
-      <AuthorSection
-        imageSource="http://i.pravatar.cc/100"
-        title="Alin Gheorghe"
-        subTitle="420 minutes ago"
-        style={{ marginTop: 15 }}
-        optionalComponent={<View />}
-      />
-      <Text p bold style={{ marginTop: 10 }}>
-        You should totally like check this out, ok? Why would you use another UI
-        library when you have so many components written by Creative Tim and the
-        whole React Native community. Galio was created by developers for
-        developers.
-      </Text>
-      <Text p style={{ marginTop: 10 }}>
-        A lot of Bacon. I'd really like to eat like a LOT of Bacon :(.
-      </Text>
-    </View>
-  </View>
+  <ImageBackground source={{ uri: bgImage }} style={{ width: '100%', height: '100%' }}>
```

Comment:
The header image shouldn't have been changed -- or at least the size and the looks of it. So... the use of <ImageBackground /> is incorrect here.